test/module/shared_model/backend_proto/common_objects/proto_common_objects_factory_test.cpp

```diff
@@ -0,0 +1,326 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+#include <ed25519/ed25519.h>
+
+#include "backend/protobuf/common_objects/amount.hpp"
```

Comment:
That header should probably be in _common_objects/common_objects_factory.hpp_ instead